Compile error when installing using pip

```
[root@foo ~]# pip install gmpy2
Collecting gmpy2
  Using cached gmpy2-2.0.8.zip
Installing collected packages: gmpy2
  Running setup.py install for gmpy2 ... error
    Complete output from command /usr/bin/python2 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-sTLF5g/gmpy2/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-eHgmfN-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_ext
    building 'gmpy2' extension
    creating build
    creating build/temp.linux-x86_64-2.7
    creating build/temp.linux-x86_64-2.7/src
    gcc -pthread -fno-strict-aliasing -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -DWITHMPFR -DWITHMPC -I/usr/include/python2.7 -c src/gmpy2.c -o build/temp.linux-x86_64-2.7/src/gmpy2.o
    In file included from src/gmpy2.c:426:0:
    src/gmpy.h:106:19: fatal error: gmp.h: No such file or directory
     #  include "gmp.h"
                       ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1

    ----------------------------------------
```

--- a/.github/workflows/build_wheels.yml
+++ b/.github/workflows/build_wheels.yml
@@ -11,7 +11,7 @@ jobs:
         os: [ubuntu-22.04, ubuntu-24.04-arm, macos-13, macos-14, macos-15, windows-2022]
     steps:
       - uses: actions/checkout@v4
-      - uses: msys2/setup-msys2@v2
+      - uses: msys2/setup-msys2@v2.27.0
         name: Setup msys2
         with:
           install: >-

--- a/docs/overview.rst
+++ b/docs/overview.rst
@@ -14,7 +14,9 @@ values.   The cutover point for performance varies, but can be as low as 20 to
     gmpy2 can crash the Python interpreter in case of memory allocation
     failure.  To mitigate this feature of memory management in the GMP library,
     you should estimate the size of all results and prevent calculations that
-    can exaust available memory.
+    can exaust available memory.  In case you are interested only in
+    the GMP bindings, you can use the
+    `python-gmp <https://pypi.org/project/python-gmp/>`_ package.
 
 The `mpfr` and `mpc` types provide support for correctly rounded multiple
 precision real and complex arithmetic via the MPFR and MPC libraries.  The

